mmapi.pm

```diff
@@ -289,4 +289,16 @@ sub wait_for_children_to_start {
     }
 }
 
+=head2 whoami
+
+    whoami();
+
+Query openqa api to retrive current jobID 
+=cut
+sub whoami {
```

Comment:
`whoami` isn't really a meaningful name, what about:
```suggestion
sub query_current_job_id {
```

Reply:
Thx @Martchus
I choose the name `get_current_job_id()` as other functions in that module are called `get_*` too, even if they query openQA as well.